### PR TITLE
Update governance.did

### PR DESCRIPTION
**Pull request by bitdivine**
# Motivation
The candid file in nns-js used to contain only a small subset of the upstream governance did file.  We will shortly need most, if not all, of the API.

# Changes
- Import the governance candid did file from the ic repo.
- Compile idl.js from this file.

# Testing
Note that MOST changes just add methods, so should be non-breaking.  There is only one breaking change:  Service now takes an argument.  It is this that needs the most careful testing.

### CODE REVIEW

File: candid/governance.did [hunk 1]
```diff
@@ -1,10 +1,336 @@
...
+  account : vec nat8;
+  joined_community_fund_timestamp_seconds : opt nat64;
+  dissolve_state : opt DissolveState;
+  followees : vec record { int32; Followees };
+  neuron_fees_e8s : nat64;
+  transfer : opt NeuronStakeTransfer;
+  known_neuron_data : opt KnownNeuronData;
+};
 type NeuronId = record { id : nat64 };
-service : {
```
Comment (bitdivine) [inline candid/governance.did:8]: service appears later - it has been moved, not deleted.

File: candid/governance.did [hunk 2]
```diff
@@ -1,10 +1,336 @@
...
+type Split = record { amount_e8s : nat64 };
+type Tally = record {
+  no : nat64;
+  yes : nat64;
+  total : nat64;
+  timestamp_seconds : nat64;
+};
+type UpdateNodeProvider = record { reward_account : opt AccountIdentifier };
+type WaitForQuietState = record { current_deadline_timestamp_seconds : nat64 };
+service : (Governance) -> {
```
Comment (bitdivine) [inline candid/governance.did:313]: Note:  Where previously we had `service {` we now have an argument.  This does not, however, seem to have had any effect on the generated js so I believe that this is safe to ignore.

Comment (ielashi) [reply candid/governance.did:313]: Yes, this has no affect at all on consumers.

File: candid/governance.did [hunk 3]
```diff
@@ -1,10 +1,336 @@
...
+  get_pending_proposals : () -> (vec ProposalInfo) query;
+  get_proposal_info : (nat64) -> (opt ProposalInfo) query;
   list_known_neurons : () -> (ListKnownNeuronsResponse) query;
-}
+  list_neurons : (ListNeurons) -> (ListNeuronsResponse) query;
+  list_proposals : (ListProposalInfo) -> (ListProposalInfoResponse) query;
+  manage_neuron : (ManageNeuron) -> (ManageNeuronResponse);
+  transfer_gtc_neuron : (NeuronId, NeuronId) -> (Result);
+  update_node_provider : (UpdateNodeProvider) -> (Result) query;
+}
```
Comment (ielashi) [inline candid/governance.did:336]: nit: new line

**Review by ielashi — APPROVED** [review]

**Comment by ielashi** [comment]
Thanks! I merged it since I'd like to build on this change.

**Comment by ielashi** [comment]
@bitdivine By the way, I think we might also need to update `governance.certified.did` as well.

**Pull request merged**